packages/upload-api/src/consumer/has.js

```diff
@@ -22,5 +22,5 @@ const has = async ({ capability }, context) => {
     )
   }
 
-  return context.models.provisions.hasStorageProvider(capability.nb.consumer)
+  return context.provisionsStorage.hasStorageProvider(capability.nb.consumer)
```

Comment:
We should checking whether `provider` matching `capability.with` is present as opposed to arbitrary one. Because one might have `nft.storage` provider but that does not allow them to store in `web3.storage` and vice versa.

I realize this is not something changes here alter, but given the migration it might be a good time to make such a change. Not a blocker.